etc/lf.nu

```diff
@@ -0,0 +1,25 @@
+# Autocompletion for nushell.
+# 
+# Documentation: https://www.nushell.sh/book/externs.html
+
+# Either add this to your existing completions or copy this to
+# your nu config file (Use 'config nu' in the nushell to open it).
+
+module completions {
```

Comment:
Is `module` required here? I thought a file already is a module by itself. For me I was able to get this to work with the following steps:

1. Remove the `module` definition and just use `export extern`
2. Save this file at `~/.config/nushell/scripts/lf.nu`
3. Load this file by adding `source lf.nu` to the config file (importing via `use` doesn't seem to work if exporting something with the same name as the module though)